x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/toolbar/components/inspect/modal.tsx

```diff
@@ -200,7 +201,9 @@ const ModalInspectQueryComponent = ({ closeModal, getInspectQuery }: ModalInspec
   return (
     <MyEuiModal onClose={closeModal} data-test-subj="modal-inspect-euiModal">
       <EuiModalHeader>
-        <EuiModalHeaderTitle>{i18n.INSPECT}</EuiModalHeaderTitle>
+        <EuiModalHeaderTitle>
+          {i18n.INSPECT} {title}
```

Comment:
I am wondering if we really the prop, can we just do that?
```suggestion
          {i18n.INSPECT} {ALERTS_TABLE_TITLE}
```

Reply:
I am wrong here!!!, if we want to re-use this component, we need to do what you did!